src/components/Sidebar/Sidebar.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from 'react';
+import './Sidebar.css';
+import { Link } from 'react-router-dom';
+
+class Sidebar extends Component {
+
+  isActive(path){
+    if (window.location.pathname !== path) {
+      return ''
+    }
+
+    return 'active'
+  }
+
+  render() {
+    return (
+      <nav className="sidenav">
+        <div className="sidebarTitle">
+          <h2>Taskhouse.io</h2>
+        </div>
+        <div className="sidebarAvatar">
+          <div className="circle"></div>
+        </div>
+        <div className="sidebarUsername">
+          <p>Frederik Kvartborg</p>
+        </div>
+        <div className="sidebarMenu">
+          <div className={["menuItem", this.isActive('/')].join(' ')}>
+            <Link to='/'><li><i className="fas fa-tasks"></i>My projects</li></Link>
```

Comment:
All menu items should be moved to an array on the class, like below.

```js
items = [
  ["My projects", "/", "fas fa-tasks"],
  ["Create Project", "/second", "fas fa-plus-square"]
]
```
Then it is possible to create a `renderItem` method which takes an item from the array and renders it, this eliminates redundant html and minimizes bugs.